### PR TITLE
login and sign up added

### PR DESCRIPTION
**Pull request by Abhinaba006**
## What is the change?
login and sign up added in the home page

## Related issue?
close: #301 

## How was it tested?
locally

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
image

### CODE REVIEW

**Comment by RushanKhan1** [comment]
Hey @Abhinaba006!
Please make the base branch of the pull request ``develop`` instead of ``master``.

File: public/css/styles.css [hunk 1]
```diff
@@ -27,6 +27,11 @@ body {
     url("https://i.pinimg.com/originals/34/cc/62/34cc62d1df1c8328f56ae7a7acf2e83a.gif")
     no-repeat center center;
 }
+
+.btn{
+  margin: 2px;
```
Comment (RushanKhan1) [inline public/css/styles.css:32]: Also, please use a relative unit( ``em`` ``vh`` ``rem`` ) rather than an absolute one.

**Comment by RushanKhan1** [comment]
I tested your code and it works as intended, so please make the changes, and then you would be good to go!

**Comment by Abhinaba006** [comment]
changes re made as asked, please review my code

**Comment by Abhinaba006** [comment]
I am sorry, I have messed up a little with git, everything is fixed and working fine now, please review my code.

**Comment by Pranav016** [comment]
What happened here :p
image

**Comment by Abhinaba006** [comment]
I was working on another issue but forget to change branch, so this happened and I have to revert back. I have squashed them, but how can I know they are successful or not?

**Comment by Pranav016** [comment]
> I was working on another issue but forget to change branch, so this happened and I have to revert back. I have squashed them, but how can I know they are successful or not?

Squashing means combining all the commits into one, removes the number of commits you chose and leaves only one commit. You can force push it into this PR so you can easily see on.github or you can check your logs

**Comment by Pranav016** [comment]
Woah, you did not have to squash all the commits, only the commits you made. Now it becomes difficult to roll back in case of production bug..

**Comment by Abhinaba006** [comment]
can I revert it? back I am having a problem with squashing?

**Comment by Pranav016** [comment]
> can I revert it? back I am having a problem with squashing?

No, squashing is permanent and I would've adviced you to work again on a new branch but you committed everything to master which is also very bad, always work on a new branch :)

**Comment by Abhinaba006** [comment]
okay, I am closing this pr, and working on a new branch

**Pull request closed without merging**